pkgs/development/libraries/libmodbus/default.nix

```diff
@@ -8,6 +8,11 @@ stdenv.mkDerivation rec {
     sha256 = "0drnil8bzd4n4qb0wv3ilm9zvypxvwmzd65w96d6kfm7x6q65j68";
   };
 
+  preConfigure = ''
+    export ac_cv_func_malloc_0_nonnull=yes
+    export ac_cv_func_realloc_0_nonnull=yes
+  '';
```

Comment:
Usually it is enough to pass this to configureFlags like this:

```
configureFlags = [
  "ac_cv_func_realloc_0_nonnull=yes"
  "ac_cv_func_malloc_0_nonnull=yes"
];
```

Reply:
yup, you were right!